content/sonar/widgets/project-coverage/description.yml

```diff
@@ -0,0 +1,4 @@
+name: Metrics
+description: Display the selected Sonar metrics of a new code in an application
```

Comment:
New widget is only related to coverage, right? If yes, update the description so it specifies it only display Sonar coverage metrics